agent/pkg/app/config.go

```diff
@@ -56,5 +57,16 @@ func configFromCLI(c *cli.Context) config.Config {
 	// postgres database
 	cfg.DB.EventEnabled = c.Bool(flagEventEnabled)
 	cfg.DB.URL = c.String(flagDBURL)
```

Comment:
Future TODO(design): we could not pass DB URL to agent when they are deployed out of us, as it would have full access to postgre, but LGTM now.

Reply:
Good idea, maybe we can create an issue.